README.md

```diff
@@ -12,6 +12,55 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
 This service provides an API, which is consumed by other CovidCertificate backend services. It offers an endpoint to allow the decoding and verification of a given Covid Certificate.
 It regularly sends a request to the [Verifier Service](https://github.com/admin-ch/CovidCertificate-App-Verifier-Service) to keep an up-to-date list of trusted DSCs with which Covid Certificates could be signed.
 
+## Simple Verification
+
+In order to allow the independent usage of the Verification-Check-Service without deeper knowledge of the source code, the repository includes a Dockerfile to start the service in a separate container. However, one needs to be in possession of an api-token to access the Verifier-Service.
+
+### Starting the service
+
+To build the image, clone the repository, `cd` into the cloned directory and run
+```bash
+docker build -t simple-verification .
```

Comment:
I would use `CovidCertificate-App-Verification-Check-Service` as name, instead of `simple-verification`